core/sbt-cloudflow/src/main/scala/cloudflow/sbt/BuildAppPlugin.scala

```diff
@@ -45,15 +45,15 @@ object BuildAppPlugin extends AutoPlugin {
     }
 
   override def projectSettings = Seq(
-    allProjectsWithCloudflowBasePlugin := Def.taskDyn {
+    allProjectDependenciesWithCloudflowBasePlugin := Def.taskDyn {
           Def.task {
-            projectWithCloudflowBasePlugin.all(ScopeFilter(inAnyProject)).value.flatten
+            projectWithCloudflowBasePlugin.all(ScopeFilter(inDependencies(ThisProject, transitive = true))).value.flatten
```

Comment:
Without this change, `buildApp` would build container images for all sub-projects.
So if there is more than 1 Application in a Blueprint, the plugin would always build all container images in the entire project - not just those of a single Application.

I assume, if the CI builds individual applications/sub-projects, the same container images should get built and published redundantly.

Reply:
Got the point regarding sbt dependencies.
So that is why `buiildApp` will just build *any* Cloudflow-related container-image on the path.

I'll try if it works without this change.

Reply:
checked with the original Filter `inAnyProject `
-> works! :)

I'll update the PR right away